src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetRemoteFileService.cs

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+using System.Runtime.Caching;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft;
+using Microsoft.ServiceHub.Framework;
+using Microsoft.ServiceHub.Framework.Services;
+using NuGet.Common;
+using NuGet.Packaging;
+using NuGet.Packaging.Core;
+using NuGet.Protocol;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    public sealed class NuGetRemoteFileService : INuGetRemoteFileService
+    {
+        private ServiceActivationOptions? _options;
+        private IServiceBroker _serviceBroker;
+        private AuthorizationServiceClient? _authorizationServiceClient;
+        private bool _disposedValue;
+
+        // TODO: right settings for cache and policy?
```

Comment:
is this todo still relevant?  if so, tracking issue URL please

Reply:
fixed in next iteration. using same settings as nugetpackagesearchservice.cs